docs/guides/developer/docs/release-manager.md

```diff
@@ -1,474 +1,366 @@
+        vim docs/guides/admin/docs/releasenotes.md
+        vim docs/guides/admin/docs/changelog.md
+        git commit -S docs/guides/admin/docs/releasenotes.md docs/guides/admin/docs/changelog.md
+        git push <remote> r/6.x
 
 3. Switch to a new branch to create the release (name does not really matter):
 
-        git checkout -b r/3.0
-
-4. Add release notes, and update the changelog using the `create-changelog`
-   [helper script](https://github.com/opencast/helper-scripts/tree/master/create-changelog) to
-   easily format the list, the commit:
-
-        vim docs/guides/admin/docs/releasenotes.md docs/guides/admin/docs/changelog.md
-        git commit docs/guides/admin/docs/releasenotes.md docs/guides/admin/docs/changelog.md -sS
-
-5. Update the [localization translations](localization.md).
+        git checkout -b tmp-6.0
 
-6. Merge release notes into release branch:
-
-        git checkout r/3.x
-        git merge r/3.0
-        git checkout r/3.0
-
-7. Make version changes for release. You can use `sed` to make things easier. Please make sure, the changes are correct:
+4. Make the version changes for the release. You can use `sed` to make things easier but please make sure that the
+   changes are correct:
 
         for i in `find . -name pom.xml`; do \
-          sed -i 's/<version>3.0-SNAPSHOT</<version>3.0</' $i; done
-
-8. Commit changes and create release tag:
-
-        git commit -asS -m 'Opencast 3.0'
-        git tag -s 3.0
-
-9. Switch back to release branch, push release notes and tags:
+          sed -i 's/<version>6-SNAPSHOT</<version>6.0</' $i; done
 
-        git checkout r/3.x
-        git push <remote> 3.0:3.0
-        git push <remote> r/3.x
+5. Have a look at the changes. Make sure no library version we use had the version `6-SNAPSHOT` and was accidentally
+   changed. Also make sure that nothing else was modified:
 
-10. You can remove the new branch afterwards:
+        git diff
+        git status | grep modified: | grep -v pom.xml   # this should have no output
 
-        git branch -D r/3.0
+6. Commit the changes and create a release tag:
 
-11. Release the branch in JIRA, and create the next one. Talk to your JIRA administrators to have this done.
+        git add $(git status | grep 'modified:.*pom.xml' | awk '{print $2;}')
+        git commit -S -m 'Opencast 6.0'
+        git tag -s 6.0
 
-12. Push the built artifacts to Maven. Bug the QA Coordinator to do this so that he remembers to set this up from the CI
-    servers.
+9. Push the tag to the community repository (you can remove the branch afterwards):
 
-13. Push the built artifacts back to GitHub. Create a new release using the
-    [graphical user interface](https://github.com/opencast/opencast/releases) to upload the distribution tarballs
-    manually.
+        git push <remote> 6.0:6.0
 
-14. Update [the website](https://github.com/opencast/opencast.github.io) with a pull request, or directly if you have
```

Comment:
Is there a reason you removed this section?  This gets missed *all* the time as is, if we don't remind RMs in the docs then it won't ever happen.

Reply:
I was surprised to find it in here at all and I wouldn't see that as part of the responsibility of a release manager to manage the website. The website is the board's responsibility from my point of view. Release managers should not have to deal with that like they also don't need to feed Twitter or Facebook.

In general, I would like to not push a lot of additional responsibilities towards release managers or else, over time it will only become harder to find volunteers for the job.

But that's something we can certainly discuss in the community. I'll bring that to list.

Reply:
Per the discussion in the technical meeting, I withdraw my objection here.  We'll figure out a way to automate this.